cmd/ocsp.go

```diff
@@ -0,0 +1,165 @@
+/*
+The MIT License (MIT)
```

Comment:
Why is this file MIT licensed? If its from another package, can't it be vendorized?